examples/next-openai-pages/pages/chat/stream-chat-attachments/index.tsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @next/next/no-img-element */
+import { useChat } from 'ai/react';
+import { useRef, useState } from 'react';
+
+export default function Page() {
+  const { messages, input, handleSubmit, handleInputChange, isLoading } =
+    useChat({
+      api: '/api/stream-chat',
```

Comment:
i'd prefer `use-chat` -- we don't have stream-chat anywhere and it does not match any concept in our code

Reply:
`/api/chat` works, so used that!